Solutions/Corvus.Tenancy.Storage.Azure.Blob/Corvus/Tenancy/Storage/Azure/TenantProviderBlobStore.cs

```diff
@@ -166,14 +166,14 @@ public async Task<ITenant> CreateChildTenantAsync(string parentTenantId)
                 Tenant child = this.serviceProvider.GetRequiredService<Tenant>();
                 child.Id = parentTenantId.CreateChildId();
 
-                // At the least, we need to copy the default blob storage settings from its parent
+                // At the least, we need to copy the tenant blob storage settings from its parent
                 // to support the tenant blob store provider. We would expect this to be overridden
                 // by clients that wanted to establish their own settings.
-                BlobStorageConfiguration? defaultStorageConfiguration = parentTenant.GetDefaultBlobStorageConfiguration();
-                child.SetDefaultBlobStorageConfiguration(defaultStorageConfiguration!);
-                if (parentTenant.HasStorageBlobConfiguration(this.ContainerDefinition))
+                BlobStorageConfiguration? defaultStorageConfiguration = parentTenant.GetBlobStorageConfiguration(ContainerDefinition);
```

Comment:
I think `defaultStorageConfiguration` might no longer be the correct name for this variable.